packages/labs/react/rollup.config.js

```diff
@@ -9,6 +9,11 @@ import {createRequire} from 'module';
 
 export default litProdConfig({
   packageName: createRequire(import.meta.url)('./package.json').name,
-  entryPoints: ['index', 'create-component', 'use-controller'],
+  entryPoints: [
+    'index',
+    'create-component',
+    'use-controller',
+    'props-without-ref',
```

Comment:
Should this be listed as an entrypoint?

Reply:
+1. We should keep exports and entrypoints in sync.

Reply:
I reverted the entrypoints so it doesn't contain the new file "props-without-ref"